skbio/io/format/genbank.py

```diff
@@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
         elif header == 'FEATURES':
             # merge the two dictionaries together. Note this is only
             # available in Python-3.5
-
             feature_metadata = merge_dicts(parsed, feature_metadata)
         else:
             metadata[header] = parsed
```

Comment:
I think it is a better idea to leave this out.

Its breaking a whole bunch of tests and I'm not entirely sure what this does.